cypress/e2e/CreateProjectButton.cy.ts

```diff
@@ -0,0 +1,6 @@
+describe('Create Project Button', () => {
+  it('Finds the Create Project Button in projects page', () => {
+    cy.visit('/projects');
+    cy.get('[data-test-id="create-project"]');
```

Comment:
plus adding the `should exist`, we can just use ids here too instead of data attribute to make it less weird looking

```suggestion
    cy.get('#create-project').should('exist');
```

Reply:
then we need to come of with a convention